KarhooUISDK/Screens/BookingRequestScreen/FormBookingRequestScreen/FormBookingRequestPresenter.swift

```diff
@@ -63,10 +63,10 @@ final class FormBookingRequestPresenter: BookingRequestPresenter {
     func load(view: BookingRequestView) {
         self.view = view
         switch Karhoo.configuration.authenticationMethod() {
-        case .guest(settings: _):
-            self.karhooUser = false
-        default:
+        case .karhooUser:
             self.karhooUser = true
+        default:
+            self.karhooUser = false
```

Comment:
@mohadian can you please confirm that token user is treated as guest and not as karhoo user? @dianakarh told me you explained to her that token user should fall under the guest user scenario

Reply:
@mohadian ok, thanks.